website/README.md

```diff
@@ -29,10 +32,36 @@ npm run build
 
 This command generates static content into the `build` directory and can be served using any static contents hosting service.
 
-## Deployment
+## Localization
 
-```console
-GIT_USER=<Your GitHub username> USE_SSH=true yarn deploy
+Localization is done on [GitLocalize](https://gitlocalize.com/repo/4999/whole_project).
+You can sign in with your GitHub account to start translating.
+
+When you add to the docs in the `mdx` files,  
+Contributors on GitLocalize will translate the added content
+and the translation will get dumped under the [i18n](i18n) folder in a later time.
+
+If you are a native speaker of the translated languages,
+and you are interested in translating your edits yourself, 
+you are welcome to navigate to the folder and do it yourself!.
```

Comment:
I think this suggestion makes huge sense. If the editor himself is a native speaker of the locale. His own translation would be more accurate, more timely, and less effortful than contributors on GitLocalize.